fiftyone/core/odm/database.py

```
@@ -284,6 +276,20 @@ def _disconnect():
     mongoengine.disconnect_all()
 
 
+def _close_async_client(client):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_close_async_client`

Add return type annotation: `None`

(ANN202)